### ISSUE TITLE
Add Navajo letterforms for smallcaps vowels with ogonek

### ISSUE THREAD

**Issue by brawer** [googlei18n/noto-fonts#982]
Noto Sans and Serif currently have no smallcaps ogonek’ed vowels in the Navajo letter form. We should add `aogonek.sc.loclNAV`, `eogonek.sc.loclNAV`, etc.

![image](https://user-images.githubusercontent.com/1527880/31700160-5a515874-b37c-11e7-83f0-b943adf8a404.png)

![image](https://user-images.githubusercontent.com/1527880/31700116-018feafc-b37c-11e7-90b8-81f0affdd626.png)

**Comment by twardoch** [comment]
Yes — in case of Navajo ogoneks, it definitely makes sense to keep making localized variants. I don't think those can be easily made "universal" akin to acutes.

**Comment by nv-wiki** [comment]
You can close this. Looks awesome, exactly as it should. Thank you.

**Comment by brawer** [comment]
No, smallcaps `aogonek.loclNAV.sc` is still missing. Likewise of the other smallcaps ogonek letters.

**Comment by nv-wiki** [comment]
yes, you are right. I did not see you meant smallcaps. 

(I guess I am just so happy there are finally some correct font sets available.)
